src/components/Search/Search.js

```diff
@@ -418,23 +418,27 @@ const Search = () => {
       })
       .then(function (json) {
         const tileBounds = setupBounds(json.bbox)
-
-        L.tileLayer(
-          `${tilerURL}/stac/tiles/{z}/{x}/{y}.png?url=${featureURL}&${tilerParams}`,
-          {
-            tileSize: 256,
-            bounds: tileBounds,
-            pane: 'imagery'
-          }
-        )
-          .addTo(clickedFootprintImageLayerRef.current)
-          .on('load', function () {
-            // hide loading spinner
-            dispatch(setSearchLoading(false))
-          })
-          .on('tileerror', function () {
-            console.log('Tile Error')
-          })
+        if (tilerURL.length > 0) {
```

Comment:
```suggestion
        if (tilerURL) {
```

an empty string is falsy, and this is easier to read